Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -80,12 +80,12 @@ def test_split_fields():
 
 
```

Comment:
why did not you add a test case for the `get_ticket_context` additional_keys+args addiiton?

Reply:
those are being tested by the modification i did in the UT 'test_command'. I can add a designated one if needed but the current one is already testing that.

Reply:
I would still suggest adding a separate test case. i'll write the docstring to explain:
given - additional keys of a ticket alongside regular keys.
when - updating a ticket
then - validate that all the details of the ticket were updated, and all the updated keys are showen in the HR and in the context.